lib/lotus/utils/path_prefix.rb

```diff
@@ -82,7 +82,7 @@ def relative_join(strings, separator = @separator)
         prefix = @string.gsub(@separator, separator)
 
         self.class.new(
-          Utils::Kernel.Array([prefix, strings]).join(separator),
+          [prefix, strings].join(separator).chomp("/"),
```

Comment:
It might be that since uniq is the breaking part of Kernel.Array here. I could keep this 'compact' and 'flattened' and just leave the 'uniq' out. Eg.
```
[prefix, strings].flatten.compact.join(separator).chomp("/")
```
What you think?